janet/src/main/java/io/techery/janet/helper/ActionStateSubscriber.java

```diff
@@ -29,6 +33,11 @@
         return this;
     }
 
+    public ActionStateSubscriber<A> onFinish(Action0 onFinish) {
```

Comment:
@almozavr This will be not so helpful, because we have onFail and onSuccess methods for this. `onFinish` is just callback to make common things for onSuccess and onFail. For example, hiding progress bar.